### PR TITLE
Opaque generation

### PR DESCRIPTION
**Pull request by spkrka**

### CODE REVIEW

**Comment by spkrka** [comment]
Ping @danielnorberg

File: src/main/java/com/spotify/folsom/client/DefaultRawMemcacheClient.java [hunk 1]
```diff
@@ -250,7 +253,12 @@ public void run() {
     public void write(final ChannelHandlerContext ctx, final Object msg,
                       final ChannelPromise promise)
         throws Exception {
-      outstanding.add((Request<?>) msg);
+      Request<?> request = (Request<?>) msg;
+      if (request instanceof BinaryRequest) {
+        ((BinaryRequest) request).setOpaque(requestSequenceId.incrementAndGet());
```
Comment (spkrka) [inline src/main/java/com/spotify/folsom/client/DefaultRawMemcacheClient.java]: I guess this one doesn't have to be an atomicInteger - is a regular integer fine, or should it be volatile?

(AFAIK, this should all run from the same thread)

Comment (danielnorberg) [inline src/main/java/com/spotify/folsom/client/DefaultRawMemcacheClient.java]: A regular integer will do fine. The connection handler is single threaded.

**Comment by danielnorberg** [comment]
:+1:

**Comment by protocol7** [comment]
+1

**Pull request merged**